content/en/docs/ambient/usage/l7-features/index.md

```diff
@@ -21,8 +23,12 @@ With a waypoint proxy deployed, you can use the following API types:
 |  Name  | Feature Status | Attachment |
 | --- | --- | --- |
 | `HTTPRoute` | Beta | `parentRefs` |
-| `TCPRoute` | Beta | `parentRefs` |
-| `TLSRoute` | Beta | `parentRefs` |
+| `TCPRoute` | Alpha | `parentRefs` |
+| `TLSRoute` | Alpha | `parentRefs` |
+
+{{< tip >}}
+`TCPRoute` and `TLSRoute` still have Alpha status in Kubernetes Gateway API. Caution is advised if considering their adoption.
+{{< /tip >}}
```

Comment:
optional, I think Lin preferred no tip so remove if Lin or yourself feel strongly about it

```suggestion
```